src/client/shadows/dynamicshadowsrender.cpp

```diff
@@ -481,11 +486,11 @@ void ShadowRenderer::renderShadowObjects(
 
 		// render other objects
 		u32 n_node_materials = shadow_node.node->getMaterialCount();
-		std::vector<s32> BufferMaterialList;
-		std::vector<std::pair<bool, bool>> BufferMaterialCullingList;
-		std::vector<video::E_BLEND_OPERATION> BufferBlendOperationList;
+		BufferMaterialList.clear();
```

Comment:
Is this necessarily faster.

Reply:
yeah you are right, did some tests and it's not very different, let's rollback this one